sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLoggingPolicy.java

```diff
@@ -43,26 +43,14 @@
     /**
      * Creates an HttpLoggingPolicy with the given log configurations.
      *
-     * @param httpLogOptions The HTTP logging configurations.
-     */
-    public HttpLoggingPolicy(HttpLogOptions httpLogOptions) {
-        this(httpLogOptions, false);
-    }
-
-    /**
-     * Creates an HttpLoggingPolicy with the given log configuration and pretty printing setting.
-     *
      * @param httpLogOptions The HTTP logging configuration options.
-     * @param prettyPrintJson If true, pretty prints JSON message bodies when logging. If the detailLevel does not
-     * include body logging, this flag does nothing.
      */
-    private HttpLoggingPolicy(HttpLogOptions httpLogOptions, boolean prettyPrintJson) {
-        this.prettyPrintJson = prettyPrintJson;
-
+    public HttpLoggingPolicy(HttpLogOptions httpLogOptions) {
         if (httpLogOptions == null) {
             this.httpLogDetailLevel = HttpLogDetailLevel.NONE;
             this.allowedHeaderNames = Collections.emptySet();
             this.allowedQueryParameterNames = Collections.emptySet();
+            this.prettyPrintJson = false;
```

Comment:
Along with the other comment, this property name could be changed to be more generic.

Reply:
Renamed this property as well.